projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py

```diff
@@ -1,75 +1,84 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
+import logging
+import os
+import uuid
+
 from airflow.providers.http.hooks.http import HttpHook
-from tenacity import sleep
-from tenacity import stop_after_attempt
+from taurus_datajob_api import ApiClient
+from taurus_datajob_api import Configuration
+from taurus_datajob_api import DataJobExecutionRequest
+from taurus_datajob_api import DataJobsExecutionApi
+from urllib3 import Retry
 from vdk.internal.control.auth.auth import Authentication
 
+log = logging.getLogger(__name__)
+
 
 class VDKHook(HttpHook):
     def __init__(
         self,
-        conn_id,
-        job_name,
-        team_name,
-        timeout=1,  # TODO: Set reasonable defaults
-        retry_limit=1,
-        retry_delay=1,
+        conn_id: str,
+        job_name: str,
+        team_name: str,
+        timeout: int = 1,  # TODO: Set reasonable default
     ):
         super().__init__(http_conn_id=conn_id)
-
         self.job_name = job_name
         self.team_name = team_name
         self.timeout = timeout
         self.deployment_id = "production"  # currently multiple deployments are not supported so this remains hardcoded
-        # hook methods will use HttpHook's `run_with_advanced_retry` method and pass the _retry_dict as a parameter
-        self._retry_dict = {
-            "stop": stop_after_attempt(retry_limit),
-            "sleep": sleep(retry_delay),
-        }
-        self.headers = {
-            "content-type": "application/json",
-            "authorization": f"Bearer {Authentication().read_access_token()}",
-        }
-
-    def start_job_execution(self) -> None:
-        """
-        API path: POST /data-jobs/for-team/{team_name}/jobs/{job_name}/deployments/{deployment_id}/executions
 
-        :return: None
+        # setting these manually to avoid using VDKConfig
+        self.op_id = os.environ.get("VDK_OP_ID_OVERRIDE", f"{uuid.uuid4().hex}"[:16])
```

Comment:
In the airflow world , those would not come from environmental variables. 

Sync with Andy or Miro (who may have more context) but generally this should be fixed in next PR